README.md

```diff
@@ -349,6 +349,7 @@ end
 ```
 
 ```ruby
+# app/guards/email_confirmation_guard.rb (or somehwere else it will be auto-loaded)
```

Comment:
```suggestion
# app/guards/email_confirmation_guard.rb
```
What do you think of just having the path here? I don't think we need to overexplain how rails handles autoload. There is also a typo in `somehwere `